### PR TITLE
Updates for the first RNAseq lecture

### PR DESCRIPTION
**Pull request by ksinghal28**
Made updates to-
1) the schedule page (I'll be doing the first lecture on Nov 27th, and My will be doing the 2nd lecture).
2) the markdown file for the first lecture that should account for the differences in running rnabio on an AMI vs running it on the students' laptops
3) uploaded lecture slides

### CODE REVIEW

File: lectures/week_10/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+```
+
+4. Enter the directory where you created the `rnabio-workspace` folder, and initialize a Docker container using the image we pulled above. `-v` tells Docker to mount our workspace directory within the Docker container as `/workspace` with read-write priveleges. You'll see in the RNAseq course `/workspace` is the base directory for nearly all commands and steps.
+
+```bash
+cd bfx-workshop/rnabio-workspace
+docker run -v $PWD/:/workspace:rw -it griffithlab/rnabio:0.0.1 /bin/bash
+```
+
+5. Use `ls` to see what's in this file, enter the `workspace` folder and then use `ls` again to see what is in the `workspace` folder.
```
Comment (chrisamiller) [inline lectures/week_10/README.md:40]:
```suggestion
5. Use `ls` to see what's in your current directory, then enter the `workspace` folder and use `ls` again to see what is in the `workspace` folder.
```

File: lectures/week_10/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+Since all the environment variables we set up for the RNA-seq workshop start with ‘RNA’ we can easily view them all by combined use of the env and grep commands as shown below. The env command shows all environment variables currently defined and the grep command identifies string matches.
+```
+env | grep RNA
+```
+
+In order to view the contents of this file, you can type:
+```
+less ~/.bashrc
+```
+To exit the file, type `q`.
```
Comment (chrisamiller) [inline lectures/week_10/README.md:89]:
```suggestion
To exit `less`, type `q`.
```

File: lectures/week_10/README.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+### Known Issues/ Discrepancies from RNAbio website
+1. When running the check strandedness tool in the Module 1, RNAseq Data section, the docker run command cannot be run from within your `griffithlab/rnabio:0.0.1` docker session. To run it, we suggest opening a new tab, `cd` into the `rnaseq` directory you created at the beginning of the course, and using the following command instead-
```
Comment (chrisamiller) [inline lectures/week_10/README.md:92]:
```suggestion
1. When running the check strandedness tool in the Module 1, RNAseq Data section, the docker run command cannot be run from within your `griffithlab/rnabio:0.0.1` docker session. To run it, we suggest that you open a new terminal window, `cd` into the `rnaseq` directory you created at the beginning of this assignment, and use the following command instead:
```

**Review by chrisamiller — APPROVED** [review]
Looks good to me - decide if my edits look okay and then we'll merge.

Next year, we should make it a goal to 
a) create a slim docker image that has the relevant tools
b) create a copy of the relevant RNAbio course sections that is modified for this course, so that we can avoid the "Known issues" section

Nice work, and thanks again!

**Comment by ksinghal28** [comment]
Thanks for the edits!
And yup, agreed that a slim docker and bfx workshop version of the RNAbio course would be better. 
I think we can compile a list of the issues once My and Mariam work through their sections and use that as a To Do list for the bfx-version branch of rnabio.

**Pull request merged**